melodies_monet/_cli.py

```diff
@@ -8,6 +8,7 @@
 from contextlib import contextmanager
 from pathlib import Path
 from typing import List, Tuple
+from monetio import models
```

Comment:
For this PR, let's change this code back to it's original coding, so remove from monetio import models in 3 locations and uncomment out the import monetio as mio line. So that in this PR this file no longer changes.

If you like Liam you can make a copy of your branch without changing these updates to make it work with Maggie's fork. But for merging into the develop branch on our official repository we do not want to include these updates.